src/SFA.DAS.AssessorService.Web/Validators/Standard/OptOutStandardVersionViewModelValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Helpers;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class OptOutStandardVersionViewModelValidator : AbstractValidator<OptOutStandardVersionViewModel>
+    {
+        public const string OneApprovedVersionRequiredMessage = "One approved standard version is required";
+
+        public OptOutStandardVersionViewModelValidator(IHttpContextAccessor httpContextAccessor, IStandardVersionClient standardVersionApiClient)
+        {
+            var epaoOrgId = EpaOrgIdFinder.GetFromClaim(httpContextAccessor);
+
+            RuleFor(vm => vm.StandardReference)
+                .NotEmpty()
+                .Must((vm, _) =>
+                {
+                    return vm != null && standardVersionApiClient.GetEpaoRegisteredStandardVersions(epaoOrgId, vm.StandardReference)
```

Comment:
The check for the vm being null is not required

Reply:
Removed